source/models/customizing-serializers.md

```diff
@@ -135,7 +136,40 @@ import DS from 'ember-data';
 export default DS.JSONSerializer.extend({});
 ```
 
-#### ID
+To change the format of the data that is sent to the backend store, you can use
+the `serialize` hook:
+
+```app/serializers/application.js
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  serialize(snapshot, options) {
+    // help! what do i do here?
+  }
+});
+```
+
+If your backend store provides data in a format other than JSON API, you can
+use the `normalizeResponse` hook to change the data into the JSON API format:
+
+```app/serializers/application.js
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
```

Comment:
You can use `normalizeResponse` to remove top-level data for server-responses. Like if your server kind of complies to JSONSerializer but puts everything in a `{ "response": ... }`, you can get do `payload = payload.response`